lib/linguist/languages.yml

```diff
@@ -2145,6 +2145,26 @@ PHP:
   aliases:
   - inc
 
+#Oracle 
+PLSQL:
+  type: programming
+  ace_mode: sql
+  tm_scope: Oracle
```

Comment:
The `tm_scope` here will need to match the one in `grammars.yml` (`source.plsql.oracle`).